examples/textConnect/config_js/config.js

```diff
@@ -120,47 +159,52 @@ jQuery.noConflict();
         // 必須項目のチェック、
         for (var b = 1; b < 6; b++) {
             if ($('#select' + b).val() !== '' && $('#copyfield1').val() === '') {
-                swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+  // swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+                swal('Error!', 'Please select a field to display the combined texts.', 'error');
                 return false;
             }
         }
         for (var c = 6; c < 11; c++) {
             if ($('#select' + c).val() !== '' && $('#copyfield2').val() === '') {
-                swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+  // swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+                swal('Error!', 'Please select a field to display the combined texts.', 'error');
                 return false;
             }
         }
         for (var d = 11; d < 16; d++) {
             if ($('#select' + d).val() !== '' && $('#copyfield3').val() === '') {
-                swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+  // swal('Error!', '「結合された文字列を表示する項目」は必須です。', 'error');
+                swal('Error!', 'Please select a field to display the combined texts.', 'error');
```

Comment:
Please use values stored in **i18n.errorMessage**, instead of only producing English messages

Reply:
I changed this part already and have committed and pushed it to this branch on 8/16. Please pull the latest commit and see changes there!